src/components/AccountDetails/index.tsx

```diff
@@ -217,24 +217,25 @@ function renderTransactions(transactions, pending) {
   return (
     <TransactionListWrapper>
       {transactions.map((hash, i) => {
-        return <Transaction key={i} hash={hash} pending={pending} />
+        return <Transaction key={i} hash={hash} pending={pending}/>
       })}
     </TransactionListWrapper>
   )
 }
 
 export default function AccountDetails({
-  toggleWalletModal,
-  pendingTransactions,
-  confirmedTransactions,
-  ENSName,
-  openOptions
-}) {
+                                         toggleWalletModal,
+                                         pendingTransactions,
+                                         confirmedTransactions,
+                                         ENSName,
+                                         openOptions
+                                       }) {
```

Comment:
is this really what prettier does? is your IDE picking up the repo's version + config?

Reply:
No, just IntelliJ. Once we have linters working for TS + a github action for it, we can reformat everything all at once